src/screens/IdentityLookUp.tsx

```diff
@@ -3,21 +3,15 @@ import { useAuth } from "react-oidc-context";
 import { jsonPrettyPrint } from "src/utils/helperFunctions/json";
 import Select, { SelectOption } from "src/components/formFields/Select";
 
-import {
-  getIdentities,
-  getIdentityDatastreamsAndRoutes,
-  Identity,
-} from "src/utils/api/identities";
+import { getIdentities, Identity } from "src/utils/api/identities";
 
 function IdentityLookUp() {
   const auth = useAuth();
   const [authToken, setAuthToken] = useState("");
 
   const [identitiesList, setIdentitiesList] = useState([]);
 
-  const [apiResponse, setApiResponse] = useState(
-    "API response will display here"
-  );
+  const [apiResponse] = useState("API response will display here");
```

Comment:
For future consideration to handle loading messages: https://react.dev/reference/react/Suspense

Reply:
This screen is not currently in use, but will consider `Suspense` down the line when refactoring.